boutdata/data.py

```diff
@@ -834,7 +834,30 @@ def recalculate_xyz(self, *,  nx=None, ny=None, nz=None):
             ]
         )
 
-    def evaluate(self, name):
+        # Also create staggered versions of the coordinates
+        self.xlow = numpy.linspace(
+            -mxg / (self.nx - 2 * mxg),
+            1.0 + (mxg - 1.0) / (self.nx - 2 * mxg),
+            self.nx,
+        )[:, numpy.newaxis, numpy.newaxis]
+        self.ylow = (
+            2.0
+            * numpy.pi
+            * numpy.linspace(
+                -myg / self.ny,
+                1.0 + (myg - 1.0) / self.ny,
+                self.ny + 2 * myg,
```

Comment:
I think this could be

```suggestion
            * numpy.linspace(
                -myg / self.ny,
                1.0,
                self.ny + 2 * myg,
                endpoint=False
```

Reply:
You would need `1 + myg / self.ny` as endpoint ...

Reply:
Good point! Ok, I'll merge as-is